src/main/java/com/ericsson/ei/controller/FailedNotificationsControllerImpl.java

```diff
@@ -69,11 +68,12 @@
             queryResponse.setQueryResponseEntity(queryResponseEntity);
             LOGGER.debug("The response is : {}", response.toString());
             if (processMissedNotification.deleteMissedNotification(subscriptionName)) {
-                LOGGER.debug("Missed notification with subscription name {} was successfully removed from database", subscriptionName);
-            }
+                LOGGER.debug("Failed notification for subscription {} was successfully removed from database",
```

Comment:
Why are we removing Failed notfication when retreiving the failed notfiication? It can be retrieved by several differents users and different times. Its better to have a TTL configured or something when Failed notification is stored in database. I can understand if/when a Subscription is removed then we can cleanup all Failed-notifications that is connected to that subscription.

Reply:
I mentioned it in my review as well and talked to Saif about this already. He said he'd remove it. There is an issue about this here as well https://github.com/eiffel-community/eiffel-intelligence/issues/354
Please assign yourself to this issue Saif and close it if you have resolved this.

Reply:
Well, I didnt remove it, as the tests are linked to it and starts failing. So I thought to create a bug for this instead as I believe it should be a separate concern. However, It was already created by you (Christoffer).

Reply:
ok.